### PR TITLE
Use box-shadow feature boxes

### LINKED ISSUE #341: Key boxes overflow incorrectly
Screenshot_20231030-224218

### PR DESCRIPTION
**Pull request by jarrodmillman**
Close #341.

### CODE REVIEW

**Comment by jarrodmillman** [comment]
2023-10-31T08:31:20,227384747-07:00

2023-10-31T08:33:57,703782225-07:00

**Comment by jarrodmillman** [comment]
2023-10-31T08:36:24,060771718-07:00

It is different than it was in v0.1, but it is more similar than v0.3 of the theme and all the text is displayed.
2023-10-31T08:39:23,349048264-07:00

File: assets/theme-css/keyfeatures.css [hunk 1]
```diff
@@ -14,12 +14,13 @@
...
-  height: 200px;
   min-width: 275px;
   width: 335px;
   margin: 25px auto;
-  border-radius: 25px;
-  border: 4px solid var(--colorBackgroundInverse);
+  border: 0.02em solid var(--pst-color-border);
+  box-shadow:
+    0 0.2rem 0.5rem var(--pst-color-shadow),
+    0 0 0.0625rem var(--pst-color-shadow) !important;
```
Comment (stefanv) [inline assets/theme-css/keyfeatures.css:23]: Why is this !important necessary?

Comment (jarrodmillman) [reply assets/theme-css/keyfeatures.css:23]: I just copied it from pst. I will investigate whether we need it.

Comment (jarrodmillman) [reply assets/theme-css/keyfeatures.css]: I removed it.

**Review by stefanv — APPROVED** [review]

**Pull request merged**